cypress/integration/examples/huge-document.ts

```diff
@@ -0,0 +1,16 @@
+describe('huge document example', () => {
+  const elements = [
+    { tag: 'h1', count: 100 },
+    { tag: 'p', count: 700 },
+  ]
+
+  beforeEach(() => {
+    cy.visit('examples/huge-document')
+  })
+
+  it('contains image', () => {
+    elements.forEach(({ tag, count }) => {
+      cy.get(tag).should('have.length', count)
+    })
+  })
```

Comment:
In this case, `elements` would only be relevant to the test block
```suggestion
  beforeEach(() => {
    cy.visit('examples/huge-document')
  })

  it('contains image', () => {
    const elements = [
      { tag: 'h1', count: 100 },
      { tag: 'p', count: 700 },
    ]

    elements.forEach(({ tag, count }) => {
      cy.get(tag).should('have.length', count)
    })
  })
```

Reply:
But it can be used in future in different tests too.